src/middlewared/middlewared/utils/directoryservices/krb5_constants.py

```diff
@@ -11,9 +8,8 @@ class KRB_Keytab(enum.Enum):
 
 
 class krb5ccache(enum.Enum):
-    SYSTEM = f'{MIDDLEWARE_RUN_DIR}/krb5cc_0'
-    TEMP = f'{MIDDLEWARE_RUN_DIR}/krb5cc_middleware_temp'
-    USER = f'{MIDDLEWARE_RUN_DIR}/krb5cc_'
+    SYSTEM = 'KEYRING:persistent:0'
+    USER = 'KEYRING:persistent:'  # middleware appends UID number to this
```

Comment:
IIRC you have the 'KEYRING:persistent:' string hardcoded elsewhere in the PR (rather than used from here).